### DISCUSSION TITLE
Changing the default favicon

### DISCUSSION THREAD

**Discussion by chrisdeeming** [postalserver/postal#2750] [Help with using Postal]
I am going to be having two separate Postal instances and I want to distinguish them each with their own favicon. 

Is this possible?

**Comment by willpower232** [comment]
You could probably use docker-compose.yml to replace the favicon.ico file in the public directory without needing to rebuild the docker image.

**Reply by chrisdeeming** [reply]
Appreciate the reply. I'm quite inexperienced with docker. Do you have any tips on how to do this? Or a guide I can reference. Thanks!

**Reply by willpower232** [reply]
Judging from the Dockerfile, the public folder is at `/opt/postal/app/public`

https://github.com/postalserver/postal/blob/2f62baa238fc1102706ee4acf079b7a876b05283/Dockerfile#L63

so in theory you could add the following to your docker-compose.yml files under the` volumes:` heading for the `web:` container

```
      - /path/to/your/favicon.ico:/opt/postal/app/public/favicon.ico
```

and then `postal stop` and `postal start` to down and up the containers (unless you just want to figure out the combination for removing and re adding only the web container)
